desktop.css

```diff
@@ -0,0 +1,375 @@
+    background-color: #3e3c3c;
+  }
+
+  .desktop-navbar {
+    display: flex;
+    gap: 8px;
+    padding: 1.5% 18% 0 10%;
+  }
+
+  .autor {
+    font-family: Inter, sans-serif;
+    font-size: 20px;
+    font-weight: 700;
+    line-height: 20px;
+    letter-spacing: 0;
+    text-align: center;
+    width: 92px;
+    height: 20px;
+    margin-right: auto;
+  }
+
+  .desk-port {
+    width: 63px;
+    height: 20px;
+  }
+
+  .desk-about {
+    width: 45px;
+    height: 20px;
+  }
+
+  .desk-contact {
+    width: 59px;
+    height: 20px;
+  }
+
+  .desk-contact,
+  .desk-about,
+  .desk-port {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .desktop-navbar a {
+    color: #fff;
+  }
+
+  .mailbox {
+    width: 24px;
+    height: 24px;
+    cursor: pointer;
+  }
+
+  /* Hero styles */
+  .sec-container {
+    width: 100%;
+    height: auto;
+    background-image: url("./desktop-img/Illustration-header.svg");
+    background-repeat: no-repeat;
+    background-size: 1003.13px 769.27px;
+    background-position-x: right;
+    background-position-y: top;
+  }
+
+  .hero {
+    display: flex;
+    flex-direction: row;
+    gap: 50px;
+  }
+
+  .desk-hero-text {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 192px 0;
+    gap: 24px;
+    width: 100%;
+    height: 739px;
+    margin-right: 51%;
+  }
+
+  .header {
+    width: 100%;
+    height: 160px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    padding-left: 0;
+  }
+
+  .sub-header {
+    width: 100%;
+    height: 24px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+    padding-left: 0%;
+    margin-left: 0;
+  }
+
+  .para {
+    width: 100%;
+    height: 123px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    padding-left: 0;
+  }
+
+  .anchor {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    order: -1;
+    width: 20px;
+    height: auto;
+    margin-left: 2%;
+  }
+
+  /* work section */
+
+  .main-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 130px 0;
+    width: 100%;
+    height: auto;
+  }
+
+  .tittle {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 24px;
+    width: 1133px;
+    height: 52px;
+    margin-top: auto;
+  }
+
+  .tittle-header {
+    width: 50%;
+    height: 52px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #091e42;
+  }
+
+  .vector {
+    width: 100%;
+  }
+
+  .work-cards {
```

Comment:
Your design implementation looks good 👍  But unfortunately, there is a new bug introduced while fixing the previous reviewer's request. Now some cards are beyond the view of the left side of the screen as you can see in the GIF below. 
[OPTIONAL] Unfortunately, the previous reviewer didn't notice that and I need to keep it optional but, not only works/projects section, almost all sections have responsive issues. Kindly fix them to have a good-looking responsive portfolio 👍 
![Screen Recording 2023-03-02 at 9 29 31 PM](https://user-images.githubusercontent.com/25887253/222521082-e4a4d35f-547d-466d-a48c-bf47822cdb45.gif)